ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/UtxoLedgerService.java

```diff
@@ -87,7 +87,8 @@ public interface UtxoLedgerService {
     UtxoFilteredTransactionBuilder filterSignedTransaction(@NotNull UtxoSignedTransaction transaction);
 
     /**
-     * Finds unconsumed states of the specified {@link ContractState} type in the vault.
+     * Finds unconsumed states and subclasses of the states of the specified {@link ContractState} type in the vault.
+     * Only use this if you really care about catching all child classes, it has poor performance.
```

Comment:
```suggestion
     * Finds unconsumed states of the specified {@link ContractState} type and its subclasses in the vault.
     * Only use this if you really care about catching all child classes. {@link findUnconsumedStatesByExactType} performs better for exact-matching use cases.
```
Sounds a bit more positive to me.